src/Facility.ConformanceApi/Answer.g.cs

```diff
@@ -17,25 +17,43 @@ namespace Facility.ConformanceApi
 	[JsonConverter(typeof(AnswerJsonConverter))]
 	public partial struct Answer : IEquatable<Answer>
 	{
+		private static readonly Dictionary<string, string> s_normalizedConstants = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
+
+		static Answer()
+		{
+			s_normalizedConstants[Strings.Yes] = Strings.Yes;
+			s_normalizedConstants[Strings.No] = Strings.No;
+			s_normalizedConstants[Strings.Maybe] = Strings.Maybe;
+		}
+
 		/// <summary>
 		/// Affirmative.
 		/// </summary>
-		public static readonly Answer Yes = new Answer("yes");
+		public static readonly Answer Yes = new Answer(Strings.Yes);
 
 		/// <summary>
 		/// Negative.
 		/// </summary>
-		public static readonly Answer No = new Answer("no");
+		public static readonly Answer No = new Answer(Strings.No);
 
 		/// <summary>
 		/// Unknown.
 		/// </summary>
-		public static readonly Answer Maybe = new Answer("maybe");
+		public static readonly Answer Maybe = new Answer(Strings.Maybe);
 
 		/// <summary>
 		/// Creates an instance.
 		/// </summary>
-		public Answer(string value) => m_value = value;
+		public Answer(string value)
+		{
+			if (!s_normalizedConstants.TryGetValue(value, out var normalizedValue))
+			{
+				normalizedValue = value.ToLowerInvariant();
```

Comment:
We should just set `m_value` to the normalized value, not also convert it to lowercase.